jobs/rocoto/atmos_products.sh

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+
+###############################################################
+## atmosphere products driver script
+## FHRLST : forecast hour list to post-process (e.g. -f001, f000, f000_f001_f002, ...)
+###############################################################
+
+# Source FV3GFS workflow modules
+. "${HOMEgfs}/ush/load_fv3gfs_modules.sh"
+status=$?
+if (( status != 0 )); then exit "${status}"; fi
+
+export job="atmos_products"
+export jobid="${job}.$$"
+
+###############################################################
+# shellcheck disable=SC2153,SC2001
+fhrlst=$(echo "${FHRLST}" | sed -e 's/f//g')  # strip off the 'f' in the forecast hour list
+IFS='_' read -ra fhrs <<< "${fhrlst}"  # convert to array
```

Comment:
Why are we including the 'f' in the list entries to begin with just to strip them off immediately? When we remove the groups I'll get rid of the 'f' as well.
```suggestion
IFS='_' read -ra fhrs <<< "${FHRLST//f}}" # strip off the 'f's and convert to array
```

Reply:
Good point.  I'll see what I can do to not include the `f` in `FHRLST`.